includes/class-coblocks-block-assets.php

```diff
@@ -291,6 +291,7 @@ public function editor_assets() {
 				'typographyControlsEnabled'      => $typography_controls_enabled,
 				'animationControlsEnabled'       => $animation_controls_enabled,
 				'localeCode'                     => get_locale(),
+				'baseApiUrl'                     => COBLOCKS_API_NAMESPACE,
```

Comment:
The key here is `baseApiUrl` but the `COBLOCKS_API_NAMESPACE` contains `coblocks`. We should either update the array key here or the key should reference an actual URL.